src/components/Dashboard.tsx

```diff
@@ -21,5 +25,35 @@ export const Dashboard: React.SFC = () => (
       </div>
     </div>
     <Transactions />
-  </div>
+  </React.Fragment>
 )
+
+export class Dashboard extends React.PureComponent<Props> {
+  constructor(props) {
+    super(props)
+  }
+
+  accountPageView = () => {
+    switch (this.props.accountPage) {
+      case AccountPageEnum.transfer: return <TransferPage />
+      case AccountPageEnum.sign: return <Sign />
+      default: return <div />
+    }
+  }
+
+  render() {
+    return (
+      <div className='vertical-spaced'>
+        <div className='columns'>
```

Comment:
I reckon you should make this a `level` component

Reply:
Looks mint